TestCases/compliance-level-3/0092-feel-lambda/0092-feel-lambda-test-01.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Contributed to DMN TCK by StrayAlien -->
+<testCases xmlns="http://www.omg.org/spec/DMN/20160719/testcase" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <modelName>0092-feel-lambda.dmn</modelName>
+    <labels>
+        <label>Compliance Level 3</label>
+        <label>FEEL Functions: lambda</label>
+    </labels>
+
+    <testCase id="001">
+        <description>A decision may return a user-defined-function (UDF) lambda</description>
+        <resultNode name="decision_001_1" type="decision">
+            <expected>
+                <value xsi:type="xsd:double">3</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="002">
+        <description>A decision may return a Function Definition lambda</description>
+        <resultNode name="decision_002_1" type="decision">
+            <expected>
+                <value xsi:type="xsd:double">4</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="003">
+        <description>A BKM may return a UDF lambda</description>
+        <resultNode name="decision_003_1" type="decision">
+            <expected>
+                <value xsi:type="xsd:double">5</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="004">
+        <description>A BKM may return a Function Definition lambda</description>
+        <resultNode name="decision_004_1" type="decision">
+            <expected>
+                <value xsi:type="xsd:double">6</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="005">
+        <description>A BKM may return a UDF lambda with a closed over formal param</description>
+        <resultNode name="decision_005_1" type="decision">
+            <expected>
+                <value xsi:type="xsd:double">20</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="005">
```

Comment:
```suggestion
    <testCase id="006">
```

Reply:
006 is missing - looks fine to me.

Reply:
As discussed in the 2021-01-15 meeting, @StrayAlien please review this code change suggested above, originally spotted by Octavian, I think it still needs to be applied.

Reply:
Thanks @tarilabs @opatrascoiu - done.

Reply:
as a thought, it might be useful to change the xsd type of the testcase id from string to ID.  But ...... that would mean quite a lot of changes to testcase ids .... I just tried it and lot of errors .... :-( maybe when we get the current PRs cleared or reduced ....